db/scripts/01_cassandra_queue.cql

```diff
@@ -13,7 +14,7 @@ CREATE TABLE queue (
 );
 
 CREATE TABLE queue_size (
```

Comment:
should we rename this to `queue_stats`, size being only one statistic we may care about